### PR TITLE
details page

### PR DESCRIPTION
**Pull request by batzionb**
implemented new design of details page
fixed cluster templates page to use new design
refactored navigation to use patternfly nav component

### CODE REVIEW

File: src/components/ClusterTemplatesPage/hack.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* Fix issue with console.page/resource/list having an extra div around the component without 100% height
+  See https://github.com/openshift/console/blob/release-4.12/frontend/public/components/resource-list.tsx#L66
+  TODO: Submit this fix to console */
+
+.co-m-list { /* stylelint-disable-line */
```
Comment (rawagner) [inline src/components/ClusterTemplatesPage/hack.css:5]: could you scope this to just our component tree ? To make sure we dont break somebody else's plugin.

something like

```suggestion
.caas-templates-page .co-m-list { /* stylelint-disable-line */
```

**Review by rawagner — CHANGES_REQUESTED** [review]

File: src/components/ClusterTemplatesPage/hack.css (hunk 1 above)
Comment (batzionb) [reply src/components/ClusterTemplatesPage/hack.css:5]: @rawagner 
I can't scope it, it's the div containing the resource/list component

**Pull request merged**